databricks_cli/sdk/permissions_service.py

```diff
@@ -0,0 +1,95 @@
+from typing import Optional
+
+from databricks_cli.sdk.preview_service import PreviewService
```

Comment:
there's no such a thing as `preview` service. please remove or rename to something like `PreviewStability`

Reply:
That's reasonable.

I'm probably going  to use the newer openapi spec that is part of the rest specification and rewrite this differently.